.github/workflows/ci-migrations.yml

```diff
@@ -0,0 +1,58 @@
+name: ci-migrations
+
+on:
+  push:
+    branches: [ main, release/** ]
+  pull_request:
+    paths:
+      - "db/**"
+  workflow_dispatch:
+
+jobs:
+  janus_migrations:
+    runs-on: ubuntu-latest
+    env:
+      BASE_SHA: "${{ github.event.pull_request.base.sha }}"
```

Comment:
I presume this will be undefined for push and workflow_dispatch events. We should either remove those triggers or skip the `git diff` based check if the pull request base is not available.

Reply:
Good point, yes the `git diff` step should be gated to just pull request.